compiler/semcall.nim

```diff
@@ -655,12 +655,7 @@ proc semResolvedCall(c: PContext, x: var TCandidate,
           else:
             x.call.add c.graph.emptyNode
         of skType:
-          var tn = newSymNode(s, n.info)
-          # this node will be used in template substitution,
-          # pretend this is an untyped node and let regular sem handle the type
-          # to prevent problems where a generic parameter is treated as a value
-          tn.typ = nil
-          x.call.add tn
+          x.call.add newSymNode(s, n.info)
```

Comment:
```suggestion
          var tn = newSymNode(s, n.info)
          # this node will be used in template substitution,
          # pretend this is an untyped node and let regular sem handle the type
          # to prevent problems where a generic parameter is treated as a value
          tn.typ = nil
          x.call.add tn
```